DeleteUserFromNodeGroup/function.json

```
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["delete"],
+      "route": "nodeGroup/{nodeGroupId}/user/{userId}"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }
+  ],
+  "scriptFile": "../dist/DeleteUserFromNodeGroup/index.js"
```

```suggestion
  "scriptFile": "../dist/RemoveUserFromNodeGroup/index.js"
```

Change in all files and directories